core/client/fs/src/main/java/alluxio/client/block/stream/DefaultBlockWorkerClient.java

```diff
@@ -135,13 +138,13 @@ public void close() throws IOException {
   @Override
   public RemoveBlockResponse removeBlock(final RemoveBlockRequest request) {
     return mBlockingStub.withDeadlineAfter(
-        Configuration.getMs(PropertyKey.USER_NETWORK_NETTY_TIMEOUT_MS), TimeUnit.MILLISECONDS)
+        Configuration.getMs(PropertyKey.USER_NETWORK_DATA_TIMEOUT_MS), TimeUnit.MILLISECONDS)
```

Comment:
Should we declare this as a static variable as well?

Reply:
Done.